components/Card.js

```diff
@@ -84,4 +84,49 @@ CardAddress.propTypes = {
   label: PropTypes.string,
 };
 
+export function PositionRequirements({ label, children }) {
+  return (
+    <div className="p-2 shaded-text" style={{ marginTop: 'auto' }}>
+      <span className="font-semibold">{label}</span>
+      <ul className="list-disc text-left pl-6">
+        {children.map(function (listItem, index) {
+          return <li key={index}>{listItem}</li>;
+        })}
+      </ul>
+    </div>
+  );
+}
+
+export function VolunteerCard({
```

Comment:
could we just skip this as this is a copy of `<Card>` and just use it like:
```
<Card>
  <PositionRequirements>sth here</PositionRequirements>
</Card>
```
?

Reply:
You're right! :woman_facepalming: Thank you, again! :grinning: